verifytree.py

```diff
@@ -47,8 +47,16 @@ def optionxform(self, optionstr):
 case_numbers = {'REPODATA': 56, 'CORE_PACKAGES': 57, 'COMPS': 58, 
                 'BOOT_IMAGES': 59}
 
-# RELAX NG schema for comps shipped with yum
-SCHEMA = '/usr/share/doc/yum/comps.rng'
+def get_schema_file():
+    """Return the path to the RELAX NG comps schema shipped with yum."""
+    base = '/usr/share/doc/yum'
+    # Our distro may use either versioned or unversioned docdirs (Fedora < 20
+    # does the former, see bug 998579), so try both variants here
+    for docs in [base, base + '-' + yum.__version__]:
+        path = docs + '/comps.rng'
+        if os.path.isfile(path):
+            return path
+    return None  # Shouldn't happen
```

Comment:
How about raising an exception instead? That should be a sufficient indicator that this shouldn't happen. Also, we could pass the list (or better yet, a tuple) of paths we've checked with the exception object, to print a more helpful message for troubleshooting.